src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final int PASSWORD_LENGTH = 10;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null
+                && repeatPassword != null
+                && password.equals(repeatPassword)
+                && password.length() >= PASSWORD_LENGTH
+                && repeatPassword.length() >= PASSWORD_LENGTH) {
```

Comment:
we don't need to check the `repeatPassword` for length - we already have `password.equals(repeatPassword)` part that handles this